WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -0,0 +1,84 @@
+<table class="table table-striped table-condensed admin-competitions">
+  <% tbody_class = title.include?("Recent") ? "show" : "hide" %>
+  <thead>
+    <tr class="admin-table-header">
+      <th>
+        <%= "#{title} (#{competitions.count})" %>
+      </th>
+      <th class="admin-delegate">
+        Delegate(s)
+      </th>
+      <th class="admin-date">
+        Date
+      </th>
+      <th class="admin-date">
+        Announced
+      </th>
+      <th class="admin-date">
+        Report posted
+      </th>
+      <th class="admin-date">
+        Results posted
+      </th>
+      <th class="admin-button">
+        <span class="hide-div">(<%= tbody_class %>)</span>
+      </th>
+    </tr>
+  </thead>
+  <tbody class="<%= tbody_class %>">
+    <% competitions.each_with_index do |competition, index| %>
+      <% announced = competition.announced_at %>
+      <% report = competition.delegate_report.posted_at %>
+      <% results = competition.results_posted_at %>
+      <tr>
+        <td>
+          <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
+          <br /><strong><%= competition.country_name %></strong>, <%= competition.cityName %>
+        </td>
+        <td class="admin-delegate">
+          <%= users_to_sentence competition.delegates %>
+        </td>
+        <td class="admin-date">
+          <%= wca_date_range(competition.start_date, competition.end_date) %>
+        </td>
+        <% if announced %>
+          <% distance_announced = (competition.start_date - announced.to_date).to_i %>
+          <% announced_class = announced_class(distance_announced) %>
+        <% end %>
+        <td class="admin-date <%= announced_class %>">
+          <% if announced %>
+            <%= announced.strftime("%b %d, %Y") %> (<%= distance_announced %>)
+          <% end %>
+        </td>
+
+        <% if report %>
+          <% distance_report = (report.to_date - competition.end_date).to_i %>
+          <% report_class = report_class(distance_report) %>
+        <% end %>
+        <td class="admin-date <%= report_class %>">
+          <% if report %>
+            <%= report.strftime("%b %d, %Y") %> (<%= distance_report %>)
+          <% end %>
+        </td>
+
+        <% if results %>
+          <% distance_results = (results.to_date - competition.end_date).to_i %>
+          <% results_class = results_class(distance_results) %>
```

Comment:
Gah, it seems like I didn't look carefully on what is it for, you haven't noticed my bad though. The variable is used as the div class so it needs to be before it.
To avoid double if I'd suggest something like this:
```ruby
<% if announced
     distance_announced = (competition.start_date - announced.to_date).to_i
     announced_class = announced_class(distance_announced)
     announced_content = "#{announced.strftime("%b %d, %Y")} (#{distance_announced})"
   end %>
<%= content_tag :td, announced_content, class: "admin-date #{announced_class}" %>
```
Please make sure that things works fine after the change.